index.js

```diff
@@ -49,6 +49,9 @@ const draggableEvents = {
 }
 const deviceEvents = isTouchDevice ? draggableEvents.touch : draggableEvents.desktop
 
+const pixelRatio = typeof window !== 'undefined' && typeof window.devicePixelRatio !== 'undefined'
+  ? window.devicePixelRatio : 1
+
```

Comment:
Also, be aware that `window.devicePixelRatio` is affected by the _zoom_ feature of browsers, meaning **its value can change at any time**. It's probably not a big deal though.